CMakeLists.txt

```diff
@@ -39,15 +47,15 @@ add_subdirectory(include)
 add_subdirectory(src)
 add_subdirectory(windows)
 if(NOT(SIMDJSON_JUST_LIBRARY))
```

Comment:
Oh, something I didn't notice before, and just FYI: you can say `if(NOT SIMDJSON_JUST_LIBRARY)` and it will work, too. Not a change you should bother with here :)

Reply:
Yeah. Good to know.